queries.sql

```diff
@@ -8,3 +8,27 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animal SET specie = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
+SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
```

Comment:
- [ ] Just as in the previous query, it would be good if we specified/classified them based on their species. KIndly update this query as well.

Reply:
Hello @Nemwel-Boniface , 
Thank you for the review but this isn't in the requirements for the task.
![requirements](https://user-images.githubusercontent.com/95056164/196705173-adc0d673-0a3e-4f52-9278-59d27b7e4d22.JPG)
In the requirements, we aren't required to do this.